api/v1alpha4/zz_generated.conversion.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
release-0.5 shouldn't have these changes since that branch is still based on go1.16

Reply:
yea nice catch thanks, fixed it now.